### DISCUSSION TITLE
content extration merges with h2 headers

### DISCUSSION THREAD

**Discussion by facundozupel** [adbar/trafilatura#612] [Q&A]
Hi! i had an issue because im trying to extract the content from blogs, but sometimes the h2 or h3 headers get attached. 
![9c078001-4ecf-46f3-90f7-6501dd48c9dd](https://github.com/adbar/trafilatura/assets/171454631/78b7d95d-8fad-4a82-867d-2364afded636)

anyone has a clue why is this happening?

**Comment by adbar** [comment]
You'd need to provide a full example, otherwise nobody can reproduce the potential bug to understand it better.

**Reply by facundozupel** [reply]
<img width="462" alt="image" src="https://github.com/adbar/trafilatura/assets/171454631/c222aa58-c410-42ef-a85e-fca4d90e7fd1">

I´m using this function to extract the content of some URLs. 

Sometimes, the trafilatura.extract() scraps the content but some rend="h2" get zipped with the <p> labels

Tell me what else do you need me to publish to help you!

**Reply by adbar** [reply]
Thanks, but this is not strictly a Trafilatura issue, you're merging its output with other elements.
